README.md

```diff
@@ -90,214 +58,220 @@ type: tutorial
+    ```
 
-- From the CockroachDB client, run the following commands:
+1. From the CockroachDB client, run the following commands:
 
-  - Create a user `maxroach` as follows:
-  <pre><code>root@example-cockroachdb-public.cockroachdb-test:26257/defaultdb> <b>CREATE USER IF NOT EXISTS maxroach;</b>
-  CREATE USER 1
+    * Create a user `maxroach` as follows:
+  
+    <pre><code>root@example-cockroachdb-public.cockroachdb-test:26257/defaultdb> <b>CREATE USER IF NOT EXISTS maxroach;</b>
+    CREATE USER 1
 
-  Time: 9.580878ms
-  </code></pre>
+    Time: 9.580878ms
+    </code></pre>
   
-  - Create a database `employees` as follows:
-  <pre><code>root@example-cockroachdb-public.cockroachdb-test:26257/defaultdb> <b>CREATE DATABASE employees;</b>
-  CREATE DATABASE
+    * Create a database `employees` as follows:
+ 
+    <pre><code>root@example-cockroachdb-public.cockroachdb-test:26257/defaultdb> <b>CREATE DATABASE employees;</b>
+    CREATE DATABASE
 
-  Time: 14.449525ms
-  </code></pre>
+    Time: 14.449525ms
+    </code></pre>
   
-  - Give our user, `maxroach`, permission to update our database, `employees` as follows:
-  <pre><code>root@example-cockroachdb-public.cockroachdb-test:26257/defaultdb> <b>GRANT ALL ON DATABASE employees TO maxroach;</b>
-  GRANT
+    * Give the user, `maxroach`, permission to update our database, `employees`, as follows:
   
-  Time: 9.308095ms
-  </code></pre>
+    <pre><code>root@example-cockroachdb-public.cockroachdb-test:26257/defaultdb> <b>GRANT ALL ON DATABASE employees TO maxroach;</b>
+    GRANT
   
-- At this point we have a `user` and a `database`.
-
-- Type `\q` to quit the client console as shown:
-<pre><code>root@example-cockroachdb-public.cockroachdb-test:26257/defaultdb> <b>\q</b>
-pod "cockroach-client" deleted
-</code></pre>
-
-### Step 3: Access the admin console on your localhost
+    Time: 9.308095ms
+    </code></pre>
+  
+    At this point, you should have a `user` and a `database`.
 
-- Let us view the results of the commands we ran in the earlier steps via the `admin console`. The console can be accessed at localhost with port forwarding.
+1. Type `\q` to quit the client console as shown:
 
-- Run the following command to port forward `8080`:
-```bash
-$ kubectl port-forward example-cockroachdb-0 8080
-```
+    <pre><code>root@example-cockroachdb-public.cockroachdb-test:26257/defaultdb> <b>\q</b>
+    pod "cockroach-client" deleted
+    </code></pre>
 
-```
-Forwarding from 127.0.0.1:8080 -> 8080
-Forwarding from [::1]:8080 -> 8080
-```
+### Step 3: Access the admin console on your localhost
 
-- Visit <http://localhost:8080> on your browser as shown. The page should load the cluster overview.
+The following steps show you how to view the results of the commands you ran in the earlier steps via the `admin console` <!--EM: is this something that the reader has to login? Is the console part of cockroachdb or RHM?-->. You can access the console at localhost with port forwarding.
```

Comment:
Yes Indeed! Remember [Step 2: Connect to the OpenShift cluster in your CLI](https://developer.ibm.com/tutorials/get-started-using-a-cockroachdb-operator-hosted-on-red-hat-marketplace/#2-connect-to-the-openshift-cluster-in-your-cli) from the base Tutorial? Since the reader will have already completed that part he will already be logged in here!

Reply:
Do you think we should link to that step? We could hyperlink "earlier steps" to it?

Reply:
Okay, do you think it's okay to link "earlier steps" to the Step 2 section in the base tutorial?

Reply:
You can just add a note if required saying:
>Note: You need to be logged in to your OpenShift Cluster with `OC Login` which you learned in [Step 2: Connect to the OpenShift cluster in your CLI](https://developer.ibm.com/tutorials/get-started-using-a-cockroachdb-operator-hosted-on-red-hat-marketplace/#2-connect-to-the-openshift-cluster-in-your-cli)